sdk/storage/Azure.Storage.Blobs/README.md

```diff
@@ -7,24 +7,35 @@ storage is optimized for storing massive amounts of unstructured data.
 Unstructured data is data that does not adhere to a particular data model or
 definition, such as text or binary data.
 
-[Source code][source] | [Package (NuGet)][package] | [API reference documentation][rest_docs] | [Product documentation][product_docs]
+[Source code][source] | [Package (NuGet)][package] | [API reference documentation][docs] | [REST API documentation][rest_docs] | [Product documentation][product_docs]
 
 ## Getting started
+
 ### Install the package
+
 Install the Azure Storage Blobs client library for .NET with [NuGet][nuget]:
 
 ```Powershell
-Install-Package Azure.Storage.Blobs
+dotnet add package Azure.Storage.Blobs --version 12.0.0-preview.1

```

Comment:
you could do `-IncludePrerelease` to avoid changing the version every time there is a new release

Reply:
Unfortunately not with `dotnet add package` and floating prerelease versions: https://github.com/NuGet/Home/issues/4699

Reply:
:| me not reading...